headless-server-commerce-app/src/main/java/com/coremedia/blueprint/boot/headlessserver/HeadlessServerCommerceApp.java

```diff
@@ -36,13 +32,13 @@ public static void main(String[] args) {
     try {
       Hooks.enable();
       SpringApplication.run(HeadlessServerCommerceApp.class, args);
-    } catch (BeanCreationException e) {
+    } catch (Exception e) {
```

Comment:
Why wasn't BeanCreationException good enough anymore?

Btw. This whole error handling seems odd. We don't have it in any other BP spring boot app. What's the point?

I know it's a copy of the BP headless server but this should just be

```
  public static void main(String[] args) {
    SpringApplication.run(HeadlessServerApp.class, args);
  }
```

Reply:
what about hooks.enable and .disable?

Reply:
Not needed. Done by HooksEnabler config in coremedia-spring-boot which should come with every app as already cap-unified-api-boot depends on it.